repositories/device.go

```diff
@@ -75,11 +80,32 @@ func (m *DeviceModel) fromModel() twomes.Device {
 }
 
 func (r *DeviceRepository) Find(device twomes.Device) (twomes.Device, error) {
+	fmt.Println("Find device", device.ID)
```

Comment:
Are you also going to remove these logs? I saw you already removed the other ones.

Reply:
That was an issue on our side, it has been removed.